ci/test-zk-64.sh

```diff
@@ -60,22 +60,21 @@ all_passed=true
 
 for file in "$BASE_DIR/cranelift/zkasm_data/generated"/*; do
   filename=$(basename -- "$file")
-
-  # it seems like zkasmtest.js don't set exit code to zero after succes 
-  # but set 1 after fail, so after fail in _should_fail_ test all tests fail
-  exit_code=0 
+  # it seems like zkasmtest sets 1 if smth goes wrong but don't set 0
+  # if everything is OK
+  exit_code=0
```

Comment:
Hm, this job is green and for first commit is red which approves my assumption. But I runned this js script right after command whith exit code 1 and exit code after script was 0, so it should be because of script provides some alternative environment??? Also, I took a look at the script, and it looks like it explicitly exit 0 if everything is OK (but I'm not an expert in js)

Reply:
Anyway I corrupted one of ``.zkasm`` and script shows fail on it and success on all before and all after this file, so this approach looks like working solution.